docs/manage/users-roles/roles/role-capabilities.md

```diff
@@ -5,379 +5,177 @@ description: Assign any of these capabilities when you create user roles.
-   </td>
-  </tr>
-  <tr>
-   <td>Manage field extraction rules
-   </td>
-   <td><a href="/docs/manage/field-extractions">Manage field extractions</a>, which speed the search process by automatically parsing fields as log messages are ingested.
-<p>Note that if you grant a role the Manage field extraction rules capability, users with that role will also have the Manage Fields, View Fields, and View field extraction rules capabilities.</p>
-   </td>
-  </tr>
-  <tr>
-   <td>View Partitions
-   </td>
-   <td>View <a href="/docs/manage/partitions-data-tiers">Partitions</a>.
-   </td>
-  </tr>
-  <tr>
-   <td>Manage Partitions
-   </td>
-   <td>View, create, edit, and delete <a href="/docs/manage/partitions-data-tiers">Partitions</a>.
-<p>Note that if you grant a role the Manage Partitions capability, users with that role will also have View Partitions and Manage S3 data forwarding capabilities.</p>
-   </td>
-  </tr>
-  <tr>
-   <td>View Scheduled Views
-   </td>
-   <td>View <a href="/docs/manage/scheduled-views">Scheduled Views</a>.
-   </td>
-  </tr>
-  <tr>
-   <td>Manage Scheduled Views
-   </td>
-   <td>View, create, edit, and delete <a href="/docs/manage/scheduled-views">Scheduled Views</a>. Note that if you grant a role the Manage Schedule View capability, users with that role will also have View Scheduled View capabilities.
-   </td>
-  </tr>
-  <tr>
-   <td>Manage S3 data forwarding
-   </td>
-   <td><a href="/docs/manage/data-forwarding/amazon-s3-bucket">Manage S3 data forwarding</a> from Sumo Logic to an S3 bucket.
-   </td>
-  </tr>
-  <tr>
-   <td>Manage Content
-   </td>
-   <td>Manage the content for your organization. This provides access to <a href="/docs/manage/content-sharing/admin-mode">Admin Mode</a> in the Library.
-   </td>
-  </tr>
-  <tr>
-   <td>Manage Tokens
-   </td>
-   <td>Manage <a href="/docs/manage/security/installation-tokens">Installation Tokens</a>.
-   </td>
-  </tr>
-  <tr>
-   <td>View Account Overview
-   </td>
-   <td>View the Account Overview page.
-   </td>
-  </tr>
-  <tr>
-   <td>View Parsers
-   </td>
-   <td>View <a href="/docs/cse/schema/parser-editor">Parsers</a>.
-   </td>
-  </tr>
-  <tr>
-   <td>Download Search Results
-   </td>
-   <td><a href="/docs/search/get-started-with-search/search-basics/export-search-results">Export</a> log query results to a .csv file.
-   </td>
-  </tr>
-</table>
-
-
+| Capability | Description |
+| :-- | :-- |
+| View Collectors	| View collectors and sources that have already been installed or added.| 
+| Manage Collectors	| [View and manage](/docs/send-data) installed and hosted collectors, as well a sources. | 
+| Manage Ingest Budgets	| Manage [ingest budgets](/docs/manage/ingestion-volume/ingest-budgets). Enabling this will automatically enable the Manage Collectors capability. The Manage Collectors capability on its own permits the re-assignment of budgets to different collectors, but not creating or deleting them.| 
```

Comment:
```suggestion
| Manage Ingest Budgets	| Allows you to manage [ingest budgets](/docs/manage/ingestion-volume/ingest-budgets). Enabling this will automatically enable the Manage Collectors capability. The Manage Collectors capability on its own permits the re-assignment of budgets to different collectors, but not creating or deleting them.| 
```